doc/glossary.rst

```diff
@@ -675,6 +675,19 @@ General Concepts
         but not maintained by the core developer team.
         See https://scikit-learn-contrib.github.io.
 
+    scikit-learn enhancement proposals
+    SLEP
+    SLEPs
+        Changes to the API principles and changes to dependencies or supported
+        versions happen via a `SLEP <governance/#enhancement-proposals-sleps>`_ and follows the
+        decision-making process outlined in the `governance document <governance>`_.
```

Comment:
Both this link and the previous one are broken in the rendered HTML:

https://54937-843222-gh.circle-artifacts.com/0/doc/glossary.html#term-slep

For the former you probably need to use:

    ... versions happen via a `SLEP <slep>`_ and follows the ...


I do not understand why the second link (targeting the "governance" reference) does not work. The `doc/governance.rst` file correctly includes the following line before its heading:

    .. _governance:

This reference is reachable as expected at:

https://54937-843222-gh.circle-artifacts.com/0/doc/governance.html#governance

and I do not see a typo in the following syntax:

    decision-making process outlined in the `governance document <governance>`_.

Maybe trying to be more explicit with the following would work?

    decision-making process outlined in the :ref:`governance document <governance>`_.

Reply:
The syntax:

    `governance document <governance>`_

I confirmed it should be:

    :ref:`governance document <governance>`

is actually wrong. I fond other occurrences of this mis-pattern in our documentation. I will issue another PR to fix those.